source/sass/blocks/specifications.scss

```
@@ -7,12 +7,20 @@
   font-size: 12px;
   line-height: 16px;
   color: $color-grey-600;
-  margin-top: 13px;
+  margin-top: 3px;
```

В Хроме у этого тега есть отступы. Надо их обнулить обязательно. Можно прямо перед заданием отступа сверху:
```suggestion
  margin: 0;
  margin-top: 3px;
```